ui/desktop/src/components/settings/session/SessionSharingSection.tsx

```diff
@@ -1,28 +1,35 @@
 import React, { useState, useEffect } from 'react';
 import { Input } from '../../ui/input';
-import { Check } from 'lucide-react';
+import { Check, Lock } from 'lucide-react';
 
 export default function SessionSharingSection() {
+  process.env.GOOSE_BASE_URL_SHARE = 'https://goosed.stage.sqprod.co/api'; // TODO: remove after testing
```

Comment:
@lily-de is there a better way to test this? i tried setting it in `main.ts` and run `just make-ui` but when i do that, i am getting `process.env.GOOSE_BASE_URL_SHARE = undefined`